esp/bindings/SOAP/Platform/soapbind.cpp

```diff
@@ -110,6 +124,7 @@ static CSoapFault* makeSoapFault(CHttpRequest* request, IMultiException* me, con
 
 int CHttpSoapBinding::onSoapRequest(CHttpRequest* request, CHttpResponse* response)
 {
+    pSoapRequestCount->inc(1);
```

Comment:
FYI onSoapRequest() is an overloaded function that has evolved over time.  It handles more than it's name implies.  The external description of this metric should really be something like JSON and SOAP POST messages.  Also, it will probably catch the majority of them but services can also capture messages at a lower level.

Reply:
The comment is fine. I think this should work. Some quirky service could bypass this somehow but most likely not any of our standard HPCC ones.

Reply:
If the comment is sufficient, let's leave it at that. Realistically, if there is a path where a request of two does not get counted, that's probably OK. However, if we find that the count is far from what is expected, we'll have to re-evaluate how this count is maintained.